### ISSUE TITLE
Google Books: handle "editor" creator

### ISSUE THREAD

**Issue by owcz** [zotero/translators#1368]
GB appears to save all authors as plain "authors" even though they are sometimes attributed as "editors" (not sure if "translators" are credited)

E.g., Lehman, David, and John Brehm. 2006. _The Oxford Book of American Poetry_. Oxford University Press. https://books.google.com/books?id=qDzKDAAAQBAJ: "edited by David Lehman, John Brehm" in left sidebar but Zotero saves both creators as "authors"

I don't know where the XML is being captured, otherwise I'd try to fix it myself

**Comment by zuphilip** [comment]
The [XML](http://books.google.com/books/feeds/volumes/qDzKDAAAQBAJ) treats them as dc:creator:
```xml
<dc:creator>David Lehman</dc:creator>
<dc:creator>John Brehm</dc:creator>
```

E.g.

> "edited by David Lehman, John Brehm" in left sidebar

I don't see that. But I have the information at the bottom: "Herausgeber: David Lehman, John Brehm". Thus, the website changes according to the user language...

**Comment by owcz** [comment]
I posted the wrong link. It shows once you pass into preview (click "Preview this book »"), though on the originally linked overview page it also reads: "Editors      David Lehman, John Brehm" 

That they are editors and not authors is an important distinction, no? If the metadata doesn't specify them as editors, though they display in the page rendering as editors, is the best method to ask Google to update on their ends or to manually scrape to detect when editors are listed?

**Comment by adam3smith** [comment]
it's always best to get better metadata rather than rely on scraping, but I'm not sure how much success we can expect here.

**Comment by zuphilip** [comment]
> That they are editors and not authors is an important distinction, no?

Absolutely, yes! But AFAIK Dublin Core `dc` has only `creator` and `contributor` role. Thus, the best we can hope for is that they will change it to `dc:contributor`, which might still not be very helpful for us. Otherwise they have to use another vocabulary.

> I posted the wrong link.

Which one would be correct link? (I am still thinking that the page might have language-agnostic parts, which make it much harder to improve scraping from, but even that could be possible...)

**Comment by owcz** [comment]
https://books.google.com/books?id=qDzKDAAAQBAJ has "Editors David Lehman, John Brehm" in the table at the bottom, and
https://books.google.com/books?id=qDzKDAAAQBAJ&printsec=frontcover (book preview page) has "edited by David Lehman, John Brehm" in the left sidebar

It appears to use the equivalent of "editors" in the French and German versions, except for the French preview page ("publié par David Lehman,John Brehm"). My French is comme ci comme ça but that isn't exactly "edited by", right?
